### ISSUE TITLE
Make to reload

### ISSUE THREAD

**Issue by golbin** [golbin/git-commander#10]
- [ ] reload function
- [ ] auto reload

**Comment by jzsfkzm** [comment]
I think watching the cwd with [chokidar](https://www.npmjs.com/package/chokidar) would help here. It's the watcher lib gulp-watch is also using.

**Comment by golbin** [comment]
@jzsfkzm Good idea. But there is a strange situation using blessed. git-commander is stuck when using async functions. And it's released when I press a key. It's why I use execSync instead of exec. It's also why I don't make push/pull yet.

I'll keep testing for this and async functions.
